src/nyancat.c

```diff
@@ -485,7 +484,7 @@ int main(int argc, char ** argv) {
 								/* This was a response to the TTYPE command, meaning
 								 * that this should be a terminal type */
 								alarm(2);
-								strcpy(term, (char *)&sb[2]);
+								term = (char *)&sb[2];
```

Comment:
I assume this is because there can be cases where the content of **sb** is modified before reaching terminal detection checks?